docs/quickstart-docker.md

```diff
@@ -0,0 +1,143 @@
+
+The following steps create a Docker volume, start Percona XtraBackup in a Docker continer, take and prepare a backup of Percona Server for MySQL database.
+{.power-number}
+
+1. Create a Docker volume
+
+    One of the benefits of using a Docker volume for persistent storage with a database is that it allows you to keep your data safe and accessible across different containers. For example, if you want to upgrade your database image to a newer version, you can stop the old container and start a new one with the same volume attached without losing any data. Another benefit is that you can quickly backup and restore your data using the docker cp command or mount the volume to another container.
+
+    ```{.bash data-prompt="$"}
+    $ docker volume create backupvol
+    ```
+    
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        backupvol
+        ```
+
+2. Start Percona XtraBackup 8.0.34 in a container, take and prepare a backup
+
+    To use the `docker run` command, specify the name or ID of the image you want to use and, optionally, some flags and arguments that modify the container's behavior. 
+
+    The command has the following options:
+
+    | Option | Description |
+    |---|---|
+    | `--name`  |  Provides a name to the container. If you do not use this option, Docker adds a random name.  |
+    | `--volumes-from` | Refers to Percona Server for MySQL and indicates that you intend to use the same data as the `psmysql` container.|
+    | `it` | Interacts with the container and be a pseudo-terminal. |
+    | `--user root` | Sets the user to root inside the container. This option is required to access the MySQL data directory and run the xtrabackup command. |
+    |`backup_8034` |Indicates the directory inside the container where the backup files are stored. |
+    | `-v` | Mounts a volume from the host machine or another container to the container that is being run. |
+    | `backupvol` | Indicates the persistent storage for the database. |
+    | `psmysql`    | Indicates the Percona Server for MySQL container.  |
+    |`--password`   | Prompts the user to enter the password for the root user. For convenience you can add the password for the root user to this option, for example, `--password=secret`. Then the password will be passed automaticaly when running the command. Note that if you specify the password with `--password=secret`, the password is visible in `docker ps`, `docker ps -a` (docker history) and regular ps command.          |
+    | `8.0.34` | Use this tag to specify a specific version. |
+
+    When running Percona XtraBackup from a container and connecting to a Percona Server container, we recommend using the `–-user root` option in the Docker command.
```

Comment:
Rewrite - we know that PXB is in a container and PS is in another container.

Reply:
done